src/builder/pod/volume_source.rs

```diff
@@ -0,0 +1,74 @@
+use k8s_openapi::api::core::v1::CSIVolumeSource;
+use std::collections::BTreeMap;
+
+#[derive(Clone)]
+pub struct SecretOperatorVolumeSourceBuilder {
```

Comment:
Looks like the rest of the `VolumeSource` stuff is in `pod::volume`, any reason not to put this there too?

Reply:
No i think we can merge volume_mount in there as well.